src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html class= "page" lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="
+            width=device-width, 
+            user-scalable=no, 
+            initial-scale=1.0, 
+            maximum-scale=1.0, 
+            minimum-scale=1.0
+          ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="iMac" class="card__img img">
```

Comment:
Why do you need this `img` class? Let's delete it
```suggestion
      <img src="./images/imac.jpeg" alt="iMac" class="card__img">
```